Dockerfile

```diff
@@ -12,6 +12,12 @@ FROM python:${PYTHON_VERSION} as run
 WORKDIR /app
 
 ENV PYTHONUNBUFFERED=1
+ENV DB_ENGINE="default"
+ENV DB_NAME="default"
+ENV DB_USER="default"
+ENV DB_PASSWORD="default"
+ENV DB_HOST="default"
+ENV DB_PORT="default"
 
 COPY --from=builder /app .
```

Comment:
The COPY command here copies everything from the builder stage to the run stage. Ensure that unnecessary files are not included in the final image to keep it lightweight. Consider using a .dockerignore file to exclude unnecessary files.